DBM-Naxx/MilitaryQuarter/Gothik.lua

```diff
@@ -76,21 +76,24 @@ local function getWaveString(wave)
 		return L.WarningWave3:format(unpack(waveInfo))
 	end
 end
-
+--]]
```

Comment:
Please only use line comments --, even for blocks of code. It is more readable in diffs. :-)